classroom-dashboard/app/controllers/sections_controller.rb

```diff
@@ -1,70 +1,69 @@
 class SectionsController < ApplicationController
+  
+  # Find the section before any other action
+  before_action :find_section, only: [:show, :edit, :update, :destroy, :flashcards]
 
-	# Find the section before any other action
-	before_action :find_section, only: [:show, :edit, :update, :destroy, :flashcards]
+  # Order in which the sections are listed on the home page
+  def index
```

Comment:
The spacing in this function appears to be inconsistent with the rest of the file.